### PR TITLE
Script to reset the games table and the elo in the user table in the database

### LINKED ISSUE #58: DB: Command to reset games and elo

### PR DESCRIPTION
**Pull request by SveaGaenzle**
I added the script `reset` in the folder `scripts` .
I also added the file `rest_database_test` in the folder `tests` for manual testing.

## Issue:

- resolves #58 

## Description:
- The method `reset_games` deletes the game table in the database.
- The table is automatically created again as soon as a new game is added.
- The method `reset_elo` resets the elo in the user database for every user to `mu`=25.000 and `sigma`=8.333.
- In the test file `reset_database_test` four new users are created and their matchmaking parameters are changed. In addition, three games are added to the game table. Then the functions `reset_games` and `reset_elo` are called and it is checked whether the matchmaking parameters of the four users now all have the default values again and whether the game table has been deleted.

## Usage:
To manual reset the game table and the elo with this script run 
```sh
python -m comprl.scripts.reset
``` 
in the terminal.

To manual test the reset-script enable the function call in the main and run
```sh
python -m tests.reset_database_test
``` 
in the terminal.

The user and game database informations can  be read from a config file or can be parsed through the command arguments.

### CODE REVIEW

File: comprl/scripts/reset.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    game_db_name = args.game_db_name or (data["game_db_name"] if data else "games")
+    user_db_path = args.user_db_path or (data["user_db_path"] if data else "data.db")
+    user_db_name = args.user_db_name or (data["user_db_name"] if data else "users")
+
+    user_answer = input(
+        "Are you sure you want to delete the games table and "
+        "reset the matchmaking parameters? (Y/N)"
+    )
+
+    if user_answer == "Y" or "y":
```
Comment (Cari1111) [inline comprl/scripts/reset.py:86]: This does not work, because it will always return true. 
Mayby use `user_answer.lower() == "y"` instead.

**Review by Cari1111 — CHANGES_REQUESTED** [review]
Thanks for this script! In general, it works really well, the user data is reset, and the game table is deleted.
The safety question does not quite work yet (see below).

File: comprl/scripts/reset.py (hunk 1 above)
Comment (SveaGaenzle) [reply comprl/scripts/reset.py]: Thank you for noticing that. I have changed it.

**Review by Cari1111 — APPROVED** [review]
The requested changes are implemented.

**Pull request merged**